dvc/stage/serialize.py

```diff
@@ -57,6 +57,7 @@ def _serialize_out(out):
     return out.def_path if not flags else {out.def_path: flags}
 
 
+@no_type_check
```

Comment:
Type checking it will be hard, at least, I don't want to attempt for now. Doing it will cause cascade of failures In `dvc/stage` module.